package.json

```diff
@@ -48,6 +48,6 @@
     "fs-extra": "^0.11.0",
     "lodash": "~2.4.1",
     "recursive-readdir": "^1.2.0",
-    "titanium": ">3.2.0"
```

Comment:
But it's not up to the user which version is installed right? With `>3.2.0` wouldn't NPM just always install the latest available?